src/Sulu/Bundle/AdminBundle/Resources/js/stores/ResourceMetadataStore/ResourceMetadataStore.js

```diff
@@ -2,25 +2,19 @@
 import Requester from '../../services/Requester';
 
 class ResourceMetadataStore {
-    // TODO load from server
-    baseUrls: {[string]: string} = {
-        snippets: '/admin/api/snippets',
-        contacts: '/admin/api/contacts',
-        accounts: '/admin/api/accounts',
-        roles: '/admin/api/roles',
-        tags: '/admin/api/tags',
-        collections: '/admin/api/collections',
-        media: '/admin/api/media',
-        pages: '/admin/api/pages',
-    };
+    endpoints: {[string]: string} = {};
+
+    setEndpoints(endpoints: {[string]: string}) {
```

Comment:
We've actually put all the variables first and the methods afterwards until now. Also, `setEndpoints` would probably make more sense right next to `getEndpoint`.